django/db/models/options.py

```diff
@@ -825,3 +824,18 @@ def _get_fields(self, forward=True, reverse=True, include_parents=True, include_
         # Store result into cache for later access
         self._get_fields_cache[cache_key] = fields
         return fields
+
+    @property
+    def has_auto_field(self):
+        warnings.warn(
+            'Model._meta.has_auto_field is deprecated. Just check if the auto_field attribute is not None instead.',
```

Comment:
RemovedInDjango21 ? I thought master is still 1.11, so the deprecation release is 1.11 and the removal is 2.0 ?